### PR TITLE
Feature/additional bug fixes

### PR DESCRIPTION
**Pull request by agduncan94**
* ng build --prod now works
* Travis will do one regular build and one ng build --prod
* Fixed footer layout no resizing
* Update login screen
* Fix search result returning to results page issue
* Fix share button formatting

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -47,10 +45,7 @@ before_script:
   - 'sleep 20'
 
 script:
-  - ng build # this does syntax checking, unit testing, and jshint
-  - ng serve --silent &
-  - ng test --watch=false --code-coverage
-  - cypress run --record --config defaultCommandTimeout=10000 #test using cypress for integration testing
+  - ./scripts/run-prod.sh
```
Comment (denis-yuen) [inline .travis.yml]: script name is a bit misleading, since it only runs prod half of the time
maybe `run-build.sh`?

File: package.json [hunk 2]
```diff
@@ -40,7 +40,7 @@
     "datatables.net-bs": "^1.10.13",
     "datatables.net-dt": "^1.10.15",
     "elasticsearch": "13.0.0",
-    "font-awesome": "^4.7.0",
+    "font-awesome": "https://registry.npmjs.org/font-awesome/-/font-awesome-4.7.0.tgz",
```
Comment (denis-yuen) [inline package.json]: A https version is a bit odd, maybe a comment in the readme?

File: scripts/run-prod.sh [hunk 3]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -ev
+
+echo "${TRAVIS_BRANCH}"
+echo "${TRAVIS_PULL_REQUEST}"
+echo "${TRAVIS_PULL_REQUEST_BRANCH}"
+echo "${TRAVIS_PULL_REQUEST_SHA}"
+
+if [ ${#RUN_PROD} == "true" ]; then
+  ng build --prod
```
Comment (denis-yuen) [inline scripts/run-prod.sh]: While we're at it, run "npm shrinkwrap" as well to ensure that that doesn't break too.

File: package.json (hunk 2 above)
Comment (agduncan94) [reply package.json]: Weird I think this might have been done automatically, I'll revert it back to 4.7.0

**Comment by garyluu** [comment]
npm install bootstrap-social?

**Pull request merged**